dict/dict0dict.cc

```diff
@@ -670,11 +659,14 @@ bool dict_table_rename_in_cache(dict_table_t *table, const char *new_name, bool
   }
 
   /* Remove table from the hash tables of tables */
-  HASH_DELETE(dict_table_t, name_hash, dict_sys->table_hash, ut_fold_string(old_name), table);
+  auto deletion_count = dict_sys->table_hash->erase(old_name);
```

Comment:
Avoid long names, I know it's considered good practice :-) but it doesn't make sense to be verbose where its obvious and scope is limited.

Reply:
Done